### PR TITLE
remove Function type.

### PR DESCRIPTION
**Pull request by sassy**
This is not good at Function type of typescript.
Fix it.

### CODE REVIEW

File: src/client.ts [hunk 1]
```diff
@@ -99,10 +98,10 @@ export class SkillConfigurator implements Clova.SkillConfigurator {
   /**
    * Create lambda handler for dispatching request.
    *
-   * @returns {Function}
+   * @returns
```
Comment (navarone-feekery) [inline src/client.ts]: Please add a return type to this function.
If there is a specific reason why no return type is included, let me know and include a reference as to why.

File: src/client.ts [hunk 2]
```diff
@@ -122,10 +121,10 @@ export class SkillConfigurator implements Clova.SkillConfigurator {
    * Create firebase handler for dispatching request.
    * However, the contents are express.
    *
-   * @returns {Function}
+   * @returns
```
Comment (navarone-feekery) [inline src/client.ts]: As above, please add a return type to this function.
If there is a specific reason why no return type is included, let me know and include a reference as to why.

**Comment by navarone-feekery** [comment]
@sassy Thanks for your contribution. 
I've noticed that your original branch is outdated. Can you please update your master branch and redo these changes?
I have also added some comments regarding function return types.

File: src/client.ts (hunk 1 above)
Comment (sassy) [reply src/client.ts]: I fixed it.

File: src/client.ts (hunk 2 above)
Comment (sassy) [reply src/client.ts]: I fixed it.

**Comment by sassy** [comment]
@navarone-feekery 
Thank you. I updated it.

**Pull request merged**